### PR TITLE
i18n.feature for UI test, hoc and frozen in spanish, portuguese, arabic

### PR DESCRIPTION
**Pull request by tanyaparker**

### CODE REVIEW

File: dashboard/test/ui/features/i18n.feature [hunk 1]
```diff
@@ -0,0 +1,44 @@
+Feature: See Hour of Code and Frozen tutorial in Spanish, Portuguese, and Arabic
+
+Background:
+  Given I am on "http://studio.code.org/reset_session"
+
+Scenario: HoC tutorial in Spanish
+  Given I am on "http://studio.code.org/s/20-hour/stage/2/puzzle/16/lang/es"
+  And I rotate to landscape
+  And I wait for 2 seconds
```
Comment (islemaster) [inline dashboard/test/ui/features/i18n.feature]: What's the purpose of this wait?  It's often more durable to use a more specific wait, like "And I wait to see {some element}".

File: dashboard/test/ui/features/i18n.feature [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+  Then element ".modal-content p:nth-child(2)" has text "Agora, vamos ver se podemos criar duas retas que formam um ângulo de 90 graus entre si. Você vai precisar usar o bloco \"vire\" e também o bloco \"mova\"."
+  And element "#prompt" has text "Agora, vamos ver se podemos criar duas retas que formam um ângulo de 90 graus entre si. Você vai precisar usar o bloco \"vire\" e também o bloco \"mova\"."
+
+Scenario: HoC tutorial in Arabic
+  Given I am on "http://studio.code.org/s/20-hour/stage/2/puzzle/16/lang/ar"
+  And I rotate to landscape
+  And I wait for 2 seconds
+  Then element ".dialog-title" has text "اللغز 16 من 20"
+  And element ".modal-content p:nth-child(2)" has text "أنا أريد زهرة عباد الشمس! إستخدم  قطعة \"اذا\" لتوصلني إلى هناك بأقل عدد من القطع."
+  And element "#prompt" has text "أنا أريد زهرة عباد الشمس! إستخدم  قطعة \"اذا\" لتوصلني إلى هناك بأقل عدد من القطع."
```
Comment (islemaster) [inline dashboard/test/ui/features/i18n.feature]: Needs EOF newline.

**Comment by islemaster** [comment]
Couple of nits, otherwise this seems great!  Glad we're going to have some localization testing.

File: dashboard/test/ui/features/i18n.feature [hunk 3]
```diff
@@ -0,0 +1,44 @@
+Feature: See Hour of Code and Frozen tutorial in Spanish, Portuguese, and Arabic
+
+Background:
+  Given I am on "http://studio.code.org/reset_session"
```
Comment (joshlory) [inline dashboard/test/ui/features/i18n.feature]: I think these links still need to be `learn.code.org` to get replaced in https://github.com/code-dot-org/code-dot-org/blob/13049e7c661f183deec9b0565d395dafeb545ca9/dashboard/test/ui/step_definitions/steps.rb#L5

File: dashboard/test/ui/features/i18n.feature [hunk 4]
```diff
@@ -0,0 +1,44 @@
...
+
+Scenario: HoC tutorial in Spanish
+  Given I am on "http://studio.code.org/s/20-hour/stage/2/puzzle/16/lang/es"
+  And I rotate to landscape
+  And I wait to see ".modal"
+  Then element ".dialog-title" has text "Puzzle 16 de 20"
+  Then element ".modal-content p:nth-child(2)" has text "¡Yo querer girasol! Usa un bloque \"si\" para llevarme allí con la menor cantidad posible de bloques."
+  And element "#prompt" has text  "¡Yo querer girasol! Usa un bloque \"si\" para llevarme allí con la menor cantidad posible de bloques."
```
Comment (joshlory) [inline dashboard/test/ui/features/i18n.feature]: Can we pull the actual value from the i18n ymls so this test doesn't break if the translations are updated?

**Comment by laurelfan** [comment]
_1875: May 20 – The Metre Convention is signed in Paris._

**Pull request merged**